src/include/robin_hood.h

```diff
@@ -188,6 +188,11 @@ static Counts& counts() {
 #    else
 #        define ROBIN_HOOD_PRIVATE_DEFINITION_HAS_NATIVE_WCHART() 0
 #    endif
+#    if _MSC_VER <= 1900
+#        define ROBIN_HOOD_BROKEN_CONSTEXPR 1
```

Comment:
I'd prefer a separate section for this, and to use the PRIVATE_DEFINITION thingy, something like this:

```cpp
#if defined(_MSC_VER) && (_MSC_VER <= 1900)
#        define ROBIN_HOOD_PRIVATE_DEFINITION_BROKEN_CONSTEXPR() 1
#else
#        define ROBIN_HOOD_PRIVATE_DEFINITION_BROKEN_CONSTEXPR() 0
#endif
```

Then the define is always available and can be used like this:

```cpp
#if !ROBIN_HOOD(BROKEN_CONSTEXPR)
    constexpr
#endif
```